hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/FileGroupReaderBasedMergeHandle.java

```diff
@@ -189,7 +190,6 @@ public void write() {
         this.insertRecordsWritten = readStats.getNumInserts();
         this.updatedRecordsWritten = readStats.getNumUpdates();
         this.recordsDeleted = readStats.getNumDeletes();
-        this.recordsWritten = readStats.getNumInserts() + readStats.getNumUpdates();
```

Comment:
`FileGroupReaderBasedAppendHandle` also does `this.recordsWritten = readStats.getNumInserts() + readStats.getNumUpdates();`.  Should that be changed now or the append handle reads log files only for log compaction so for append handle the record written is still correctly derived?

Reply:
This can be removed, the count is already updated in the `doAppend -> writeToBuffer -> bufferRecord -> bufferInsertAndUpdate`